src/scenes/home/index.jsx

```diff
@@ -98,6 +106,27 @@ export const Home = () => {
     }
 
   }
+  
+  const [modalIsOpen, setIsOpen] = React.useState(false);
```

Comment:
For consistency, let's:
- put this at the top where all the  other states are set up
- call `useState` rather than `React.useState`, since that's what the other lines are doing.